modules/ROOT/partials/module-loading/bundling-plugin-language-files.adoc

```diff
@@ -0,0 +1,45 @@
+== Plugin language files
```

Comment:
Should include a quick link to this file in the note for plugin language file. Suggestion:
```
The plugin language files (such as ./plugins/plugin/langs/sv_SE.js) are required where the editor user interface is localized using the [language option](http://localhost:4000/tinymce/6/ui-localization/#language). Please prefer to Plugin language files
```

Reply:
@tiny-ben-tran added in `last` commit.